src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -732,19 +731,15 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         if (!_profile.HasBellSound())
         {
-            std::vector<hstring> newSounds;
+            std::vector<IMediaResource> newSounds;
             if (const auto inheritedSounds = _profile.BellSound())
             {
-                // copy inherited bell sounds to the current layer
-                newSounds.reserve(inheritedSounds.Size());
-                for (const auto sound : inheritedSounds)
-                {
-                    newSounds.push_back(sound);
-                }
+                newSounds.resize(inheritedSounds.Size()); // fill with null
+                inheritedSounds.GetMany(0, newSounds);

```

Comment:
This could use `wil::to_vector` instead. We use it in 3 places already (and hopefully more soon).